Gordon360/Controllers/RecIM/ActivitiesController.cs

```diff
@@ -1,23 +1,23 @@
 ﻿using Gordon360.Models.CCT;
 using Gordon360.Services;
-using Gordon360.Services.RecIMServices;
+using Gordon360.Services.RecIM;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Data.SqlClient;
 using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using IActivityService = Gordon360.Services.RecIM.IActivityService;
```

Comment:
This shouldn't be necessary here, since you aren't `using Gordon360.Services`, there should be no ambiguity